pkg/tempo/config.go

```diff
@@ -202,21 +202,17 @@ func (c *RemoteWriteConfig) UnmarshalYAML(unmarshal func(interface{}) error) err
 type SpanMetricsConfig struct {
 	LatencyHistogramBuckets []time.Duration                  `yaml:"latency_histogram_buckets,omitempty"`
 	Dimensions              []spanmetricsprocessor.Dimension `yaml:"dimensions,omitempty"`
-
-	// MetricsExporter is a Prometheus metrics exporter
-	MetricsExporter metricsExporterConfig `yaml:"metrics_exporter,omitempty"`
+	// Namespace if set, exports metrics under the provided value.
+	Namespace string `yaml:"namespace,omitempty"`
+	// ConstLabels are values that are applied for every exported metric.
+	ConstLabels map[string]interface{} `yaml:"const_labels,omitempty"`
+	// Exporter is the configuration to export the metrics
+	Exporter metricsExporterConfig `yaml:"exporter"`
```

Comment:
unless we think we will add more configuration in `metricsExporterConfig` I don't think we should add this extra config level

Reply:
Right, the exporter will be configured via the prometheus instance. Removed in 521f7c346f2370b3d71de07e078d7e6d8c84face